src/app/core/entity/entity-type-selector/entity-type-selector.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit, Input } from "@angular/core";
+import { BasicAutocompleteComponent } from "../../common-components/basic-autocomplete/basic-autocomplete.component";
+import { MatFormField } from "@angular/material/form-field";
+import { FormControl, ReactiveFormsModule } from "@angular/forms";
+import { EntityConstructor } from "../model/entity";
+import { EntityRegistry } from "../database-entity.decorator";
+
+/**
+ * Component for selecting an entity type from a dropdown.
+ */
+@Component({
+  selector: "app-entity-type-selector",
+  templateUrl: "./entity-type-selector.component.html",
+  imports: [BasicAutocompleteComponent, MatFormField, ReactiveFormsModule],
+  standalone: true,
+})
+export class EntityTypeSelectorComponent implements OnInit {
```

Comment:
I would like to name this "EntityTypeSelect" not ...selector because we also have an EntitySelectComponent. So we don't introduce different terms for similar functions